eventmesh/Etcd3EventMeshServer.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const config = require('./config');
+const logger = require('../common/logger');
```

Comment:
`common/logger.js` imports `common/config.js` which loads config using `SETTINGS_PATH` env. Ideally it should use `SF_EVENTMESH_SETTINGS_PATH`

Reply:
As discussed.
We can segregate the tests similar to this: https://github.com/amitmalav/service-fabrik-broker/tree/integrate/d-h